src/xrpld/ledger/ApplyViewImpl.h

```diff
@@ -86,6 +122,8 @@ class ApplyViewImpl final : public detail::ApplyViewBase
 
 private:
     std::optional<STAmount> deliver_;
+    std::vector<STObject> batchExecution_;
```

Comment:
```suggestion
    std::vector<STObject> batchExecutions_;
```
And should update this variable anywhere it's used as well.